corehq/apps/data_cleaning/views/main.py

```diff
@@ -37,14 +40,25 @@ class CleanCasesSessionView(BaseProjectDataView):
     urlname = "data_cleaning_cases_session"
     template_name = "data_cleaning/clean_cases_session.html"
 
+    def get(self, request, *args, **kwargs):
+        try:
+            return super(CleanCasesSessionView, self).get(request, *args, **kwargs)
```

Comment:
Does this need `(CleanCasesSessionView, self)` or can it just be called as `super()`?